<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatatableDialog</class>
 <widget class="QDialog" name="DatatableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="CommonTitleBar" name="top_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#top_widget{
background-color:rgb(7,128,237);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottom_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#bottom_widget{background-color:rgb(7,128,237);}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="back_widget" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#back_widget{background-color:rgb(242,250,255);border-radius:3px;}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">QLabel{font:14px &quot;微软雅黑&quot;; color:rgb(51,51,51);}</string>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>26</number>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>25</number>
             </attribute>
             <column>
              <property name="text">
               <string>时间</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>值</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtondata">
             <property name="styleSheet">
              <string notr="true">QPushButton#pushButtondata{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(148,210,255), stop:1 rgb(4,136,255));
	border:1px solid rgb(7,128,237);
	border-radius:4px;
	font:14px &quot;微软雅黑&quot;;
	color:white;
	min-height:30px;
	max-height:30px;
	min-width:114px;
	max-width:114px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonTitleBar</class>
   <extends>QWidget</extends>
   <header location="global">guibaselib/commontitlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
